test/sqlite/index.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const sqlite = require("sqlite3");
+
+async function main() {
+  const db = new sqlite.Database(":memory:");
+
+  const aCallback = (err) => console.log("Err", err);
+
+  // We are serializing the commands just to get the same [id, parent_id] pairs
+  // for the events in the appmap snapshot in every test run.
```

Comment:
This might have this effect, but it makes the resulting appmap very weird due to all the statements being run serially after the `serialize` block finishes. Instead you can use `promisify` and `await` every call to make sure they return before calling the next one.